Source/WebCore/platform/LayoutUnit.h

```diff
@@ -59,10 +59,10 @@ while (0)
 
 #endif
 
-static const int kLayoutUnitFractionalBits = 6;
+static constexpr int kLayoutUnitFractionalBits = 6;
 static constexpr int kFixedPointDenominator = 1 << kLayoutUnitFractionalBits;
-const int intMaxForLayoutUnit = INT_MAX / kFixedPointDenominator;
```

Comment:
I don't think the `static` are needed here.

Reply:
@weinig - Done.

Reply:
@weinig - ping?